app/controllers/application_controller.rb

```diff
@@ -34,7 +34,10 @@ def user_for_paper_trail
   end
 
   def default_url_options
-    { host: ENV.fetch('HOST', 'localhost:3000') }
+    {
```

Comment:
Did you mean to include this as part of the commit?

Reply:
Yes. This allows anyone to choose a particular host they want from an environment variable HOST.  I thought that the default host was 'localhost:3000'.  If that's not an accurate default, we can change that to another host.

Reply:
I believe the default is `localhost:3000` - if that's the case, you have an incorrect port number below.

Reply:
Ah yes, I see.